resources/monitoring/templates/grafana/dashboards-1.14/etcd.yaml

```diff
@@ -1035,9 +1036,7 @@ data:
         "schemaVersion": 13,
         "sharedCrosshair": false,
         "style": "dark",
-        "tags": [
-            "kubernetes-mixin"
```

Comment:
Please add this back to all the dashboards. We added this tag to show that these dashboards belong to kubernetes-mixin.

Reply:
Hm, could you elaborate what this tag actually means? I carried over the dashboards from the upstream charts. Most of them have this tag, but some don't.

Reply:
Re-added the missing tags